doc/api/vm.md

```diff
@@ -14,6 +14,8 @@ const vm = require('vm');
 JavaScript code can be compiled and run immediately or compiled, saved, and run
 later.
 
+*Note*: The vm module makes a wrong impression as a security mechanism. In fact, it is not at all, you shouldn't use it to run untrusted scripts.
```

Comment:
Please line wrap at 80 chars.
Also, please avoid the use of informal pronouns like `you`.